hr_gamification_email_notification/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "OSI HR Gamification Extended",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators",
+    "summary": """Send Email to all the Employees with all the award given in
+     the last week.""",
+    "category": "Helpdesk",
+    "maintainers": "Open Source Integrators",
```

Comment:
This key is incorrect. It must be a GitHub user, and with a list, not a simple string.